### DISCUSSION TITLE
Hi 👋 New Updrafter here! We're all early... What brought you to Cyfrin Updraft?

### DISCUSSION THREAD

**Discussion by tippi-fifestarr** [Cyfrin/foundry-full-course-cu#1195] [Polls]
Just started watching the welcome and came curious to the Discussions tab.

As a Chainlink Developer Expert, I got Early Access a couple days ago, but didn't see it until I finally checked my emails after the Constellation Hackathon.  Would love to know what communities and interest brought other frenz

**Comment by Hemanthpradeep** [comment]
Hi there !
Even i saw the mail lately and hooked to the videos of PC !

Good to know you ! I am just a Techical enthusiast trying to learn Solidity and Blockchain !

**Comment by sagoto80** [comment]
can anyone help on this https://github.com/Cyfrin/foundry-full-course-f23/discussions/1181

**Comment by debsouryadatta** [comment]
From Youtube yeah!🙌

**Comment by CadetTalha** [comment]
from yt
